ui/src/parseError.ts

```diff
@@ -12,12 +12,15 @@ import { localize } from './localize';
 export function parseError(error: any): IParsedError {
     let errorType: string = '';
     let message: string = '';
+    let stack: string | undefined;
 
     if (typeof (error) === 'object' && error !== null) {
         if (error.constructor !== Object) {
             errorType = error.constructor.name;
         }
 
+        stack = error.stack;
```

Comment:
My PR removes the first part of the path that may contain username. How about I just add that functionality after you merge?

Reply:
Sounds good